blueocean-dashboard/src/main/js/components/RunDetailsHeader.jsx

```diff
@@ -155,8 +155,21 @@ class RunDetailsHeader extends Component {
             </div>
         );
 
-        const causeMessage = (run && run.causes.length > 0 && run.causes[run.causes.length - 1].shortDescription) || null;
```

Comment:
@imeredith I think you should split this out into a `CauseCell` component that gets reused in both the header and within the `RunMessageCell`.

@kzantow causes have `_class` - should we be making the rendering for different causes an extension point?

Reply:
Agree, we should absolutely not have teh same code in two places.

Reply:
Am doing that now